assets/src/scripts/wdfn-home/usmap/index.js

```diff
@@ -53,12 +53,10 @@ const usMap = function(node, {latitude, longitude, zoom}, store) {
         markerGroup.addTo(map);
 
         features.forEach(f => {
-            if (f.geometry) {
+            if (f.geometry && f.properties) {
                 const marker = L.circle(f.geometry.coordinates.reverse(), {
-                    color: 'red',
-                    fillColor: '#f03',
-                    fillOpacity: 0.2,
-                    radius: 5000
+                    radius: 5000,
+                    className: `site-marker ${f.properties.MonitoringLocationIdentifier}`
```

Comment:
Adding this in here to support another stream of work to allow interactions between the site summaries in the sidebar and the map markers.